convert2rhel/pkghandler.py

```diff
@@ -1043,25 +1055,29 @@ def _get_packages_to_update_yum():
     return all_packages
 
 
-def _get_packages_to_update_dnf():
+def _get_packages_to_update_dnf(reposdir):
     """Query all the packages with dnf that has an update pending on the
     system."""
-    from dnf import Base  # pylint: disable=E0401
+    import dnf  # pylint: disable=E0401
 
     packages = []
-    base = Base()
-    # Fix the case when we are trying to query packages in Oracle Linux 8
-    # when the DNF API gets called, those variables are not populated by default.
-    if system_info.id == "oracle":
-        base.conf.substitutions["ocidomain"] = "oracle.com"
-        base.conf.substitutions["ociregion"] = ""
-
-    # Same thing for CentOS, but this time, the URL for the vault has changed
-    # and instead of just ussing $releasever (8.5, 8.4...), we need either
-    # using the correct $releasever (8.5.2111) or setting up the $contentdir
-    elif system_info.id == "centos":
-        base.conf.substitutions["contentdir"] = system_info.id
```

Comment:
This was only a temporary solution to fix #356. We replaced it with a better way of expanding the variables on the system. 

Basically, on lines 1079-1080, we expand the variables that are located under `/etc/dnf/vars`.